integration/broken_subscription_test.go

```diff
@@ -0,0 +1,120 @@
+package integration_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	k8sApiErrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	addonsv1alpha1 "github.com/openshift/addon-operator/apis/addons/v1alpha1"
+	"github.com/openshift/addon-operator/integration"
+)
+
+func TestAddon_BrokenSubscription(t *testing.T) {
```

Comment:
Would you mind adding a little doc block explaining what "BrokenSubscription" means?
Especially when we test failure cases we should document the context and intend of the test somewhat detailed.

Reply:
good catch! :) done